src/ai/backend/client/cli/params.py

```diff
@@ -55,6 +57,30 @@ def convert(self, value, param, ctx):
         return value
 
 
+class StorageProxyAddressParamCheckType(click.ParamType):
+    name = "storage-proxy-address-check"
+
+    def convert(self, value: Union[str, Mapping[str, str]], param, ctx) -> Mapping[str, str]:
+        if isinstance(value, dict):
+            return value
+        if not isinstance(value, str):
+            self.fail(
+                f"expected string, got {value!r} of type {type(value).__name__}",
+                param, ctx,
+            )
+        override_map = {}
+        for assignment in value.split(","):
+            try:
+                x, y = assignment.split("=")
```

Comment:
Let add `maxsplit=1` to limit the number of parsed segments to prevent errors.
Let's use `key` and `value` (or `k` and `v`) as more sensible variable names instead of `x` and `y`. (`x` and `y` looks like position/location/coordinate stuffs).

FYI: You may also use [`str.partition()` method](https://docs.python.org/3.10/library/stdtypes.html#str.partition) interchangeably with `str.split(..., maxsplit=1)`.

Reply:
Also let's check if the key or the value is empty or not.